src/test/java/uk/gov/hmcts/reform/hmc/utils/TestingUtil.java

```diff
@@ -635,7 +635,8 @@ public static HearingEntity getCaseHearingsEntity() {
         hearingEntity.setCaseHearingRequests(List.of(caseHearingRequestEntityWithPartyOrg()));
 
         hearingEntity.getCaseHearingRequests().get(0).setVersionNumber(1);
-        hearingEntity.getCaseHearingRequests().get(0).setHearingParties(Arrays.asList(hearingPartyEntityOrg()));
+        hearingEntity.getCaseHearingRequests().get(0).setHearingParties(Arrays.asList(
```

Comment:
Again, not your change, but for consistency in this method, use `List.of(...)` instead of `Arrays.asList`

Reply:
Updated in https://github.com/hmcts/hmc-cft-hearing-service/commit/f258ba42a8e3a5c96b9786cca080d3344e0f71c8